core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/ContainerPool.scala

```diff
@@ -195,8 +243,10 @@ class ContainerPool(childFactory: ActorRefFactory => ActorRef,
               // Add this request to the buffer, as it is not there yet.
               runBuffer = runBuffer.enqueue(r)
             }
-            // As this request is the first one in the buffer, try again to execute it.
-            self ! Run(r.action, r.msg, retryLogDeadline)
+            if (!poolConfig.clusterManagedResources) {
```

Comment:
> //cannot do this in cluster managed resources, since it will introduce a tight loop

Why the distinction here between cluster managed and non cluster managed case? Would not the tight loop would exist for both cases?

Reply:
Agreed this seems wrong, just wasn't sure which cases are broken there. Removed it for now, will revisit if this is an issue for merging.